docs/src/modules/components/AppFrame.js

```diff
@@ -98,6 +99,24 @@ const styles = theme => ({
     marginLeft: theme.spacing(2),
     flex: '0 1 auto',
   },
+  skipNav: {
+    position: 'absolute',
+    padding: theme.spacing(1),
+    backgroundColor: theme.palette.background.paper,
+    transition: theme.transitions.create('top', {
```

Comment:
Are you happy with the animation? Something feels off. I can't tell if it's the duration that is too slow or the easing curve that is weird.

Reply:
Ah, yes. I'd forgotten you have to apply them "backwards" (exit transition goes on focus style, and vice versa).